### ISSUE TITLE
Running Website Mode from the command line?

### ISSUE THREAD

**Issue by dbateyko** [micahflee/onionshare#1065]
Is it possible to run website mode from the command line, and if so, could someone please add documentation to the wiki? Thanks!

**Comment by mig5** [comment]
@dbateyko yes, you can run `onionshare --website /path/to/folder/containing/your/html/site`. We'll get this added to the wiki, cheers.

**Comment by hiromipaw** [comment]
@mig5 I'll add it to the wiki.

**Comment by hiromipaw** [comment]
Added to the wiki.

**Comment by xaloc33** [comment]
Hi! How can I enable persistent mode from the command line? Thanks :)
